phongo_compat.h

```diff
@@ -147,7 +136,7 @@
 # define phongo_long long
 # define PHONGO_LONG_FORMAT "ld"
 # define SIZEOF_PHONGO_LONG SIZEOF_LONG
-# define phongo_str(str) str
+# define ZSTR_VAL(str) str
```

Comment:
A number of the exception messages for PHPC-741 involved reporting `zend_class_entry->name`. On PHP 7, this required accessing `zend_string->val`. Digging a little deeper, I found that we were inconsistently using either the `phongo_str()` macro or using `#if` blocks to conditionally use `ZSTR_VAL()` for PHP 7. Simply backporting `ZSTR_VAL()` to PHP 5.x allowed me to remove our custom macro and delete those `#if` blocks.